pkcs12/pkcs12.go

```diff
@@ -306,6 +346,66 @@ func DecodeAllCerts(pfxData []byte, password string) (certificates []*x509.Certi
 	return
 }
 
+// DecodeCodesignCertificates returns an array of certificate and private keys
+// Used to parse p12 file containing multiple codesign identities (exported from macOS Keychain)
+func DecodeCodesignCertificates(pfxData []byte, password string) (identities []SigningIdentity, err error) {
```

Comment:
DecodeAll or DecodeAllCertsAndPrivateKeys

Reply:
remove DecodeAllCerts

Reply:
is not it equivalent to `DecodeChain `: https://github.com/bitrise-io/go-utils/pull/94/files#diff-daede74dc1a2aed976de72338be4792dR250

Reply:
no